src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -489,7 +489,7 @@ namespace TerminalCoreUnitTests
             term.SetSelectionEnd({ 21, 10 });
 
             // Validate selection area
-            ValidateLinearSelection(term, { 4, 10 }, { 32, 10 });
+            ValidateLinearSelection(term, { 4, 10 }, { gsl::narrow<til::CoordType>(4 + text.size()), 10 });

```

Comment:
FWIW i'd rather not have our tests do math to calculate where things end up. Reading this diff line, I can't tell you if that's `33` (for example), which is the original value plus one (to make it exclusive).

IMO tests like this should make a developer express their gut feelings ("this selection MUST end at 33") rather than a self-referential outcome ("this selection MUST end where the length of the text suggests it would end")